### PR TITLE
Deduplicate suite.py

### PR DESCRIPTION
**Pull request by pitr-ch**

### CODE REVIEW

File: mx.truffleruby/suite.py [hunk 1]
```diff
@@ -4,15 +4,6 @@
...
-                "name": "truffle",
-                "subdir": True,
-                "version": "ac78c6f6119bed0175c07ed732b3e12b689b5d1c",
-                "urls": [
-                    {"url": "https://github.com/oracle/graal.git", "kind": "git"},
-                    {"url": "https://curio.ssw.jku.at/nexus/content/repositories/snapshots", "kind": "binary"},
-                ]
-            },
             {
                 "name": "tools",
```
Comment (eregon) [inline mx.truffleruby/suite.py]: It might be worth a comment here that `tools` imports `truffle`.

**Comment by eregon** [comment]
This can be reopened since we go with #1257

**Comment by eregon** [comment]
@pitr-ch Could you add the comment?

File: mx.truffleruby/suite.py [hunk 2]
```diff
@@ -4,15 +4,7 @@
...
-            {
-                "name": "truffle",
-                "subdir": True,
-                "version": "ac78c6f6119bed0175c07ed732b3e12b689b5d1c",
-                "urls": [
-                    {"url": "https://github.com/oracle/graal.git", "kind": "git"},
-                    {"url": "https://curio.ssw.jku.at/nexus/content/repositories/snapshots", "kind": "binary"},
-                ]
-            },
+            # Import only tools which depends on truffle
```
Comment (eregon) [inline mx.truffleruby/suite.py]: Minor: I would add ` to avoid duplication` to clarify there is technical problem to import both.

Comment (pitr-ch) [reply mx.truffleruby/suite.py]: the comment exists only until #1258, not improving further

**Pull request merged**